libs/minecraft/AreaParallelUtils.arucas

```diff
@@ -0,0 +1,134 @@
+import * from minecraft.PosUtils;
```

Comment:
What does `AreaParallelUtils` even mean? You could come up with a better name then that